core/domain/exp_jobs_one_off_test.py

```diff
@@ -414,6 +416,183 @@ def test_nonhuman_committers_not_counted(self):
             feconf.MIGRATION_BOT_USERNAME, exploration_summary.contributor_ids)
 
 
+class ExplorationContributorsSummaryOneOffJobTest(test_utils.GenericTestBase):
+    ONE_OFF_JOB_MANAGERS_FOR_TESTS = [
+        exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob]
+
+    EXP_ID = 'exp_id'
+
+    USERNAME_A = 'usernamea'
+    USERNAME_B = 'usernameb'
+    EMAIL_A = 'emaila@example.com'
+    EMAIL_B = 'emailb@example.com'
+
+    def setUp(self):
+        super(ExplorationContributorsSummaryOneOffJobTest, self).setUp()
+        self.signup(self.EMAIL_A, self.USERNAME_A)
+        self.signup(self.EMAIL_B, self.USERNAME_B)
+        self.process_and_flush_pending_tasks()
```

Comment:
Hm, I think the only other place we use it is in dispatching events from the handlers in event_services -- we stick them into a deferred task queue. But I don't understand why they're being used in e.g. OneOffReindexExplorationsJobTest, either. If it doesn't break anything, it's fine to remove it.

FYI, you could check -- using self.count_jobs_in_taskqueue (defined in core.tests.test_utils) -- whether there's anything in the taskqueue; if not, please feel free to delete that call. It might possibly have been due to an earlier event which doesn't exist any more.